<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EstoqueSimples</class>
 <widget class="QMainWindow" name="EstoqueSimples">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Controle de Estoque Simples</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbTituloClientes">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>20</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(85, 170, 255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Total de Clientes</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbTotalClientes">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
background-color: rgb(128, 194, 255);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbTituloProdutos">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>20</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(69, 209, 128);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Total de Produtos</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbTotaInformatica">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 127);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbTituloInformatica">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>20</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(69, 209, 128);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Informatica</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbTituloBebidas">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>20</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(69, 209, 128);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Bebidas</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbTotaIBebidas">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 127);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbTituloCongelados">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>20</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(69, 209, 128);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Congelados</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbTotaICongelados">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 127);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbTituloUtilidades">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>20</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(69, 209, 128);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Utilidades</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbTotaIUtilidades">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 127);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbTituloFrutas">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>20</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(69, 209, 128);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Frutas</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbTotaIFrutas">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 127);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>613</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 85, 85);
color: rgb(255, 255, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
selection-color: rgb(123, 255, 138);
selection-background-color: rgb(12, 12, 12);</string>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuClientes">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Clientes</string>
    </property>
    <addaction name="actionCadastrarCliente"/>
    <addaction name="actionVerClientes"/>
   </widget>
   <widget class="QMenu" name="menuProdutos">
    <property name="title">
     <string>Produtos</string>
    </property>
    <addaction name="actionCadastrarProduto"/>
    <addaction name="actionVerProdutos"/>
   </widget>
   <addaction name="menuProdutos"/>
   <addaction name="menuClientes"/>
  </widget>
  <action name="actionClientes">
   <property name="text">
    <string>Clientes</string>
   </property>
  </action>
  <action name="actionProdutos">
   <property name="text">
    <string>Produtos</string>
   </property>
  </action>
  <action name="actionClientes_2">
   <property name="text">
    <string>Clientes</string>
   </property>
  </action>
  <action name="actionProdutos_2">
   <property name="text">
    <string>Produtos</string>
   </property>
  </action>
  <action name="actionCadastrarCliente">
   <property name="text">
    <string>Cadastrar Cliente</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionVerClientes">
   <property name="text">
    <string>Ver Clientes</string>
   </property>
  </action>
  <action name="actionCadastrarProduto">
   <property name="text">
    <string>Cadastrar Produto</string>
   </property>
  </action>
  <action name="actionVerProdutos">
   <property name="text">
    <string>Ver Produtos</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
